test/PackageVersionsTest/InstallerTest.php

```diff
@@ -89,6 +89,7 @@ public function testDumpVersionsClass()
 
         $expectedPath = $vendorDir . '/ocramius/package-versions/src/PackageVersions';
 
+        /** @noinspection MkdirRaceConditionInspection */
```

Comment:
@Ocramius : I'd just ignore test-classes in the inspection, bunch of inspection are doing ths already.